engine/src/field.h

```diff
@@ -721,6 +725,9 @@ class MCField : public MCControl, public MCMixinObjectHandle<MCField>
     void GetPageHeights(MCExecContext& ctxt, uindex_t& r_count, uinteger_t*& r_heights);
     void GetPageRanges(MCExecContext& ctxt, MCInterfaceFieldRanges& r_ranges);
     
+    void GetKeyboardType(MCExecContext& ctxt, intenum_t& r_type);
```

Comment:
These remain as intenum_t as they are bound to the property setting logic (which doesn't understand explicit enum types).